tap_hubspot/hubspot.py

```diff
@@ -172,7 +172,7 @@ def get_deals_v2(
         )
 
         for chunk in chunker(gen, 100):
-            ids: List[str] = [contact["id"] for contact in chunk]
+            ids: List[str] = [deal["id"] for deal in chunk]
```

Comment:
this got introduced by this PR for some reason

Reply:
Has been changed in latest commit on this PR

Reply:
could you rebase it to completely remove it from history?